src/components/admin/UserProfileModal/components/header/ProfileModalEditBasicInfo.tsx

```diff
@@ -61,7 +58,7 @@ export const ProfileModalEditBasicInfo: React.FC<ProfileModalEditBasicInfoProps>
         pictureUrl={pictureUrl}
         register={register}
       />
-      <div className="ProfileModalEditBasicInfo__default-avatars">
+      <div className="ProfileModalEditBasicInfo__default-avatars mb-2">
```

Comment:
Same here for the `.scss` file classNames.

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8